src/SUMMARY.md

```diff
@@ -103,6 +101,14 @@
 
 - [Special types and traits](special-types-and-traits.md)
 
+- [Names](names.md)
+    - [Namespaces](names/namespaces.md)
+    - [Scopes](names/scopes.md)
+    - [Preludes](names/preludes.md)
+    - [Paths](paths.md)
+    - [Name resolution](names/name-resolution.md)
```

Comment:
I think "name resolution" is a common term when discussing compilers. I've only ever seen "name lookup" used when discussing C++, since that's what the spec writers decided to use. I don't think it is super important, since both phrases will be understood, but I would lean towards the one that is used more often for clarity. Examples:

Python: https://docs.python.org/3/reference/executionmodel.html#resolution-of-names
Ada: http://www.ada-auth.org/standards/12rm/html/RM-4-1.html
PHP: https://www.php.net/manual/en/language.namespaces.rules.php
EcmaScript: https://262.ecma-international.org/9.0/index.html#sec-lexical-environment-and-environment-record-specification-types
PL/SQL: https://docs.oracle.com/cd/A87860_01/doc/appdev.817/a77069/d_names.htm
C++ spec uses "name resolution" in the context of template processing.
https://en.wikipedia.org/wiki/Name_resolution_(programming_languages)
A Theory of Name Resolution: https://web.cecs.pdx.edu/~apt/esop15.pdf
Declarative Name Binding and Scope Rules: https://gkonat.github.io/assets/publication/declarative_name_binding_and_scope_rules-sle12.pdf
Cooper & Torczon "Engineering a Compiler" 2nd ed p. 258

Reply:
"Resolution" also sounds like a more "involved" process than just looking the name up, that may include things like fixed-point iteration (which is needed to resolve imports).

Reply:
Alright. If everybody else also uses it, that's fine. I'm putting it on record that I still don't like the term.